src/Utils/HtmlExtractorUtil.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+namespace LearnosityQti\Utils;
+
+/**
+ * Description of HtmlExtractorUtil
```

Comment:
Can you provide a meaningful description of this class here? Just a simple one-line description of what this class is or what it does, and maybe the nature of it (e.g. mentioning that it is a static / stateless class).

Reply:
Done